### PR TITLE
Upload fix for action-bar

### PR DESCRIPTION
**Pull request by pbhatnagar3**
@percyliang @kashizui please review

### CODE REVIEW

File: codalab/apps/web/bundles.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ def upload_bundle(self, source_file, bundle_type, worksheet_uuid):
             '''
             # Construct info for creating the bundle.
             bundle_subclass = get_bundle_subclass(bundle_type) # program or data
-            metadata = metadata_util.request_missing_metadata(bundle_subclass, {}, initial_metadata={'name': source_file.name, 'description': 'Upload ' + source_file.name})
+            metadata = metadata_util.request_missing_metadata(bundle_subclass, {}, initial_metadata={'name': source_file.name, 'description': 'Upload ' + source_file.name}, bypass_editor=True)
```
Comment (percyliang) [inline codalab/apps/web/bundles.py]: Let's split request_missing_metadata into two functions, one for filling in default values, and one for actually requesting missing values.  Then, here we should just call the former.  In bundle_cli.py, we should call possibly both functions.

**Comment by pbhatnagar3** [comment]
Put out a request for updated codalab-cli with two functions:
fill_missing_metadata(...)
request_missing_metadata(...) 

and updated this branch with fill_missing_metadata

@percyliang @kashizui please review

**Comment by kashizui** [comment]
LGTM 👌

**Comment by percyliang** [comment]
LGTM

**Pull request merged**